clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -494,4 +495,53 @@ public static String readFileAsString(String path, Charset charset) throws IOExc
     public static String readFileAsString(String path) throws IOException {
         return Utils.readFileAsString(path, Charset.defaultCharset());
     }
+
+    /**
+     * Creates a set
+     * @param elems the elements
+     * @param <T> the type of element
+     * @return Set
+     */
+    public static <T> HashSet<T> mkSet(T... elems) {
+        HashSet<T> set = new HashSet<T>();
```

Comment:
Do you mean "better" to be more readable? Or more efficient? The current code is does less work than `new HashSet<>(Arrays.asList(elems))`.